pkg/local_object_storage/metabase/status.go

```diff
@@ -1,14 +1,23 @@
 package meta
 
 import (
-	"errors"
+	"fmt"
 
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
 	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	"go.etcd.io/bbolt"
 )
 
+// BucketValue pairs a bucket index and a value that relates
+// an object.
+type BucketValue struct {
+	BucketIndex int
```

Comment:
this is metabase version-specific info, right? i'd also return the version in `ObjectStatus`

Reply:
> this is metabase version-specific info, right?

yes, but we have never changed buckets, only extend them. fixed